test/fixtures/rules/std_rule_cases/L039.yml

```diff
@@ -112,3 +112,75 @@ test_redshift_at_time_zone:
   configs:
     core:
       dialect: redshift
+
+test_excess_space_without_align_alias:
+  fail_str: |
+        SELECT
+            a    AS first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a AS first_column,
+        b AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo
+  configs:
+    rules:
+      L039:
+        align_alias: false
+
+test_excess_space_with_align_alias:
+  fail_str: |
+        SELECT
+            a    AS first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a           AS first_column,
+        b           AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo
+  configs:
+    rules:
+      L039:
+        align_alias: true
+
+test_missing_keyword_with_align_alias:
+  fail_str: |
+        SELECT
+            a    first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a            first_column,
+        b           AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo
+  configs:
+    rules:
+      L039:
+        align_alias: true
+
+test_skip_alias_with_align_alias:
+  fail_str: |
+        SELECT
+            a   ,
+            b   ,
+            (a   +   b) /   2
+        FROM foo
+  fix_str: |
+    SELECT
+        a ,
+        b ,
+        (a + b) / 2
+    FROM foo
```

Comment:
@barrywhart I added 2/3 tests to `test/fixtures/rules/std_rule_cases/L039.yml`, since my code shouldn't have any impact on `Line 149` (and I wouldn't know how to cover that)

1. `test_missing_keyword_with_align_alias` to cover `Line 228`, aka that we appropriately `pad` for `AS` (since `L011` will add the `AS` when `aliasing: explicit`)
2. `test_skip_alias_with_align_alias` to cover `Line 198`, aka we skip `aligning aliases` when there are no `aliases` mentioned in a `select clause`